website/docs/r/nat_gateway.html.markdown

```diff
@@ -0,0 +1,49 @@
+---

```

Comment:
This file will need to be added to the website index ERB

Reply:
Done